docs/node/run-your-node/prerequisites/stake-requirements.md

```diff
@@ -1,59 +1,66 @@
+|                             | Mainnet  | Testnet  |
+| --------------------------- | -------- | -------- |
+| Entity Registration         | 100 ROSE | 100 TEST |
+| Validator Node Registration | 100 ROSE | 100 TEST |
+| Validator Set Size          | 120      | 110      |
 
-For more information about obtaining information on your entity's account, see
+If you want to be part of the active validator set, you will need enough
+ROSE/TEST staked --  this means you must be one of the top 120 entities (respectively top 110 on the testnet)
+by stake. You can check the current top validators on blockchain explorers
+like [Oasis Scan].
+
+For more information on obtaining information about your entity's account, see
 the [Account Get Info] doc.
 
-Staking thresholds may change in the future. Read the
+Please note that staking thresholds may change in the future. Refer to the
 [native token information] to see the current values used by the network.
 
-[staking thresholds]: ../../genesis-doc.md#staking-thresholds
-[Oasis Scan]: https://www.oasisscan.com/validators
-[Account Get Info]: ../../../general/manage-tokens/cli/account.md#show
-[native token information]: ../../../general/manage-tokens/cli/network.md#show-native-token
+For the testnet, you can receive a limited number of TEST tokens by using our [Oasis Network Testnet Faucet][faucet-testnet].
+For more tokens please contact us on our official [Discord in the #testnet channel][discord].
 
 :::info
 
 To determine if you are eligible to receive a delegation from the Oasis Protocol
 Foundation, see the [Delegation Policy] document.
 
-[Delegation Policy]: ../../../get-involved/delegation-policy.md
-
 :::
 
 :::info
 
 The size of the consensus committee (i.e. the validator set) is configured by
-the [**max_validators** consensus parameter].
-
-[**max_validators** consensus parameter]: ../../genesis-doc.md#consensus
+the [**max_validators** consensus parameter][**max_validators** consensus parameter].
 
 :::
 
-## Testnet
+## ParaTime
 
-For the Testnet you are going to need TEST tokens. You can receive a limited
-number of TEST by using our [Oasis Network Testnet Faucet][faucet-testnet]. For
-more tokens please contact us at our official [Discord #testnet
-channel][discord-testnet].
+To run a ParaTime compute node, the minimum you need to stake currently is:
 
+|            | Mainnet     | Testnet  |
+| ---------- | ----------- | -------- |
+| Sapphire   | 50,000 ROSE | 100 TEST |
+| Emerald    | 50,000 ROSE | 100 TEST |
+| Cipher     | 100 ROSE    | 100 TEST |
+
+You can receive a limited number of TEST tokens by using our [Oasis Network Testnet Faucet][faucet-testnet].
+For more tokens please contact us on our official [Discord in the #testnet channel][discord].
+
+[Oasis Scan]: https://www.oasisscan.com/validators
+[Account Get Info]: ../../../general/manage-tokens/cli/account.md#show
+[native token information]: ../../../general/manage-tokens/cli/network.md#show-native-token
```

Comment:
Move these at the end of the `## Consensus` section since that's where they are used.

```suggestion
[Oasis Scan]: https://www.oasisscan.com/validators
[`oasis account show`]: ../../../general/manage-tokens/cli/account.md#show
[`oasis network show native-token`]: ../../../general/manage-tokens/cli/network.md#show-native-token
```